.github/workflows/build-oca-image.yml

```diff
@@ -0,0 +1,73 @@
+name: Build
+
+on:
+  pull_request:
+    branches:
+      - "14.0*"
+  push:
+    branches:
+      - "14.0"
+      - "14.0-ocabot-*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v4
+
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      -
+        name: Build and push
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: Dockerfile
+          tags: |
+            ghcr.io/oca/oca-custom:14.0-oca-latest
+          # cache-from: type=registry,ref=ghcr.io/oca/oca-custom:14.0-oca-latest
+          cache-to: type=local,dest=/tmp/.buildx-cache
+          # pull: true
+          load: true
+      -
+        name: Test install oca_all in docker image
+        run: |
+          docker compose up -d db
+          sleep 10 # Attendre que PostgreSQL soit prêt
+          docker compose run --rm odoo odoo -c /etc/odoo.cfg -i oca_all --without-demo= --stop-after-init
+
+      -
+        name: Login to ghcr.io
+        if: ${{ github.repository_owner == 'OCA' && github.ref == 'refs/heads/14.0' || startsWith(github.ref, 'refs/tags/') }}
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@sbidoul I suppose some github configuration  are require here if we want to push the docker image in the github registry !

Reply:
I think actions have write access to the registry by default.